linkerd/docs/client_tls.md

```diff
@@ -87,3 +87,44 @@ routers:
           certPath: /certificates/cert.pem
           keyPath: /certificates/key.pem
 ```
+
+### Client TLS and transformers
+
+When using any kind of transformer with client TLS using `io.l5d.static`, ensure that the client
+TLS's `prefix` field starts with at least the first transformer that is applied to a client name in 
+the dtab. This will ensure that TLS is initiated between the Linkerd client and any downstream 
+service receiving the intended TLS traffic. For example given a configuration:
+
+```yaml
+routers:
+- protocol: http
+  interpreter:
+      kind: default
+      transformers:
+      - kind: io.l5d.port
+        port: 4141
+  dtab: | 
+    /svc => /#/io.l5d.fs;
+  client:
+    kind: io.l5d.static
+    configs:
+    - prefix: /#/io.l5d.fs/{service}
```

Comment:
Should this be `/%/io.l5d.port/4141/#/io.l5d.fs/{service}`?

Reply:
That is correct. I will make update the config example.

Reply:
A strange thing encountered was that the client name in the UI did not have a beginning forward slash. The user that reported this issue mentioned that they added a prefix that began with `%/`. From my understanding, it should start with the `%/` but I am not entirely sure.